support/dspace_interface/client.py

```diff
@@ -671,9 +671,21 @@ def create_bitstream(self, bundle=None, name=None, path=None, mime=None,
         payload = {'properties': json.dumps(properties) + ';application/json'}
         h = self.session.headers
         h.update({'Content-Encoding': 'gzip'})
-        req = Request('POST', url, data=payload, headers=h, files=files)
-        prepared_req = self.session.prepare_request(req)
-        r = self.session.send(prepared_req)
+
+        mp_encoder = MultipartEncoder(
+            fields={
+                'file': (name, open(path, 'rb'), 'text/plain'),
```

Comment:
encoding is missing, you sure text/plain is ok?

Reply:
Encoding info is not needed there. I removed that `text/plain`. My mistake.